engine/gui/src/gui_private.h

```diff
@@ -105,72 +105,65 @@ namespace dmGui
 
     struct Node
     {
-        dmVMath::Vector4    m_Properties[PROPERTY_COUNT];
-        dmVMath::Vector4    m_ResetPointProperties[PROPERTY_COUNT];
-        dmVMath::Matrix4    m_LocalTransform;
-        dmVMath::Vector4    m_LocalAdjustScale;
-        uint32_t            m_ResetPointState;
+        dmVMath::Matrix4        m_LocalTransform;
+        dmVMath::Vector4        m_Properties[PROPERTY_COUNT];
+        dmVMath::Vector4        m_LocalAdjustScale;
```

Comment:
Fyi: While biggest-first is a common approach, they all have the same alignment (vector4), so it wasn't strictly necessary to switch these around.